src/components/homePage/filmSearch/search/Search.tsx

```diff
@@ -1,17 +1,20 @@
 import styles from "./search.module.scss";
 import SearchArrow from "../../../ui/icon/SearchArrow";
 import { useRef } from "react";
+import classNames from "classnames";
 
-interface searchProps {
+interface SearchProps {
   searchText: string;
   onSearchChange: (text: string) => void;
   onSearchClickButton: () => void;
+  theme:string
```

Comment:
`theme?: string`
please make in all components props theme as non required prop

Reply:
+